app/utils/url.js

```diff
@@ -122,6 +124,16 @@ export function matchDeepLink(url, serverURL, siteURL) {
     return null;
 }
 
+export function channelIsReachable(state, channelName, teamName) {
```

Comment:
I think you can convert this to a selector instead, thoughts?

Reply:
Gave this a shot for a bit, but ran into some trouble and tabling it for as a fast-follow refactor.

You're right, and also this logic's home shouldn't be amongst URL helper functions. I got a bit stuck and didn't want this to hold up delivery.

Reply:
Done.